juju/juju
# [JUJU-2241] Update cmd version to 3.0.4

This PR updates the cmd package version to 3.0.4 to add the new `documentation` command.

## Checklist

*If an item is not applicable, use `~strikethrough~`.*

~- [ ] Code style: imports ordered, good names, simple structure, etc~
~- [ ] Comments saying why design decisions were made~
~- [ ] Go unit tests, with comments saying what you're testing~
~- [ ] [Integration tests](https://github.com/juju/juju/tree/develop/tests), with comments saying what you're testing~
~- [ ] [doc.go](https://discourse.charmhub.io/t/readme-in-packages/451) added or updated in changed packages~

## QA steps

Try the `juju documentation` command and check that it displays a markdown summary with all the existing help.
